philosophy/FIELDS.md

```diff
@@ -0,0 +1,104 @@
+# Fields
+
+Fields combine a label, an input element and valdiation messages.
+
+Think of Fields as a form of API sugar expressed as additional components.
```

Comment:
~~API sugar~~ syntactic sugar?

Reply:
To me "API sugar" is closer to what we mean. Syntactic sugar is something like `foo?.bar` instead of having to write `foo && foo.bar`.

Here it's more about providing a nicer API / set of components to work with. I dunno, it's just an analogy anyways so it's not too important.

Reply:
Sure, but syntactic sugar is an expression, API sugar isn't. 
[ref](https://www.google.com/search?q=api+sugar&rlz=1C5CHFA_enDE799DE799&oq=api+sugar&aqs=chrome..69i57j69i60j69i61j69i60j69i61j0.825j0j7&sourceid=chrome&ie=UTF-8)

Reply:
......but, .........but it could be. Let a boy dream 👦 :P

Let's change it to syntactic sugar then. I really don't mind either way.